### ISSUE TITLE
How detach\remove\destroy?

### ISSUE THREAD

**Issue by kqlek** [Reflejo/jquery-countdown#37]
Hi, how detach\remove\destroy countdown and create new?
I am trying to do a countdown, and at an interval to synchronize with the server

No work:
$('.nowdigits').unbind().detach();
After creating a counter buggy

**Comment by kqlek** [comment]
I solved the problem by setting the trigger and cleaning cycles ...
But it may have a built-in option?
